app/backend/src/couchers/servicers/public.py

```diff
@@ -107,3 +107,36 @@ def GetPublicUser(self, request, context, session):
                     badges=[badge.badge_id for badge in user.badges],
                 )
             )
+
+    def GetSignupPageInfo(self, request, context, session):
+        # last user who signed up
+        last_signup, geom = session.execute(
+            select(User.joined, User.geom).where(User.is_visible).order_by(User.id.desc())
```

Comment:
Is there no `created_at` column on the User table? I guess ids are reliable but that seems even more so?